tests/ras/test_sspl.py

```diff
@@ -896,11 +883,8 @@ def test_4361(self):
         response = self.alert_api_obj.generate_alert(
             AlertType.CONTROLLER_FAULT_RESOLVED)
 
-        assert response[0], "{} {}".format(response[1],
-                                           "Couldn't connect to port "
-                                           "7900. Please try on other "
-                                           "controller (Generating "
-                                           "Fault-resolved)")
+        assert response[0], f"{response[1]} Couldn't connect to port 7900. Please try on other " \
```

Comment:
Can we use our lib assert, if its not a big change ?

Reply:
Doesn't make any difference. These are older files, it will be big change if we want to use assert utils at all places.
Keeping as is for now.